example/resnet/resnet.go

```diff
@@ -185,11 +206,12 @@ func main() {
 	initializer.ManualSeed(1)
 	trainTar := flag.String("data", "/tmp/imagenet_training_shuffled.tar.gz", "data tarball")
 	testTar := flag.String("test", "/tmp/imagenet_testing_shuffled.tar.gz", "data tarball")
+	label := flag.String("label", "", "label vocabulary")
```

Comment:
The name `label` is confusion for users, maybe `vocab`?

Reply:
Will fix it in next PR